lib/services/azurepostgresqldb/cmd-bind.js

```diff
@@ -0,0 +1,74 @@
+/* jshint camelcase: false */
+
+'use strict';
+
+var async = require('async');
+var common = require('../../common');
+var Config = require('./service');
+var log = common.getLogger(Config.name);
+
+var postgresqldbBind = function (params) {
+    var provisioningResult = params.provisioning_result;
+
+    this.bind = function (postgresqldbOperations, next) {
```

Comment:
In my test, the integration test can't pass. I think because it doesn't call `postgresqldbOperations.setParameters` here. Did you run the integration test?

Reply:
Sorry about that, I thought I did! I will have a look now.

Reply:
I found the issue in my side. I didn't run `npm install` to install the new dependency `pq`. Rerunning!